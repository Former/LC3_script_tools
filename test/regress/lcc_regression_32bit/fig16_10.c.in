10
912345
8
9
7
5
454321
3
7
0

