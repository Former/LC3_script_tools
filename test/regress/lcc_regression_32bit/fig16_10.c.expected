Enter 10 numbers.
Input number 0 : 10
Input number 1 : 912345Input number 2 : 
8
Input number 3 : 9
Input number 4 : 7
Input number 5 : 5
Input number 6 : 454321Input number 7 : 
3
Input number 8 : 7
Input number 9 : 0
The average of these numbers is 136671
